server/controller/requirement.controller.js

```diff
@@ -64,15 +67,15 @@ requirementController.post(
  * POST/
  * Create all new requirements if not exist, otherwise update
  */
-requirementController.post('/all', (req, res) => {
-  req.body.forEach(requirement => {
+requirementController.post('/all', async (req, res) => {
```

Comment:
**Problem**: some courses appear twice (in GE, and in major requirement) in the JSON list. Hence, it will create 2 same courses with 2 different types.

**Solution**: Adding `await`